realm/realm-library/src/androidTest/kotlin/io/realm/entities/SetContainerClass.kt

```diff
@@ -26,7 +26,7 @@ import java.util.*
 
 open class SetContainerClass : RealmObject() {
 
-//    val myRealmModelSet = RealmSet<StringOnly>()
+    val myRealmModelSet = RealmSet<DogPrimaryKey>()
```

Comment:
Maybe worth also to have some tests for objects without primary key, just to show that we are not using `equalsTo` and catch any accidentals regressions on that matter.

Reply:
I've added a test to cover it.